awx/main/signals.py

```diff
@@ -422,6 +422,7 @@ def emit_activity_stream_change(instance):
             object1=instance.object1,
             object2=instance.object2,
             summary_fields=summary_fields,
+            volume_tag='activity_stream',
```

Comment:
these logs are still not ideal, because they only contain the _message_ without the _data_, but I will leave the formatting problems associated with that for another day.